modulefiles/gfsutils_gaeac6.intel.lua

```diff
@@ -0,0 +1,32 @@
+help([[
+  This module loads libraries required for building and running GFS UTILS 
+  on the NOAA RDHPC machine Gaea C6 using Intel-2023.1.0.
+]])
+
+whatis([===[Loads libraries needed for building the UFS Weather Model on Gaea C6 ]===])
+
+prepend_path("MODULEPATH", "/ncrc/proj/epic/spack-stack/c6/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/ncrc/proj/epic/spack-stack/c6/spack-stack-1.6.0/envs/gsi-addon/install/modulefiles/Core")
+
+stack_intel_ver=os.getenv("stack_intel_ver") or "2023.2.0"
+load(pathJoin("stack-intel", stack_intel_ver))
+
+stack_cray_mpich_ver=os.getenv("stack_cray_mpich_ver") or "8.1.29"
+load(pathJoin("stack-cray-mpich", stack_cray_mpich_ver))
+
+stack_python_ver=os.getenv("stack_python_ver") or "3.11.6"
+load(pathJoin("stack-python", stack_python_ver))
```

Comment:
What is using python in this repo?

Reply:
@aerorahul I removed the python load and retested the C6 build successfully.